ISM43362/ISM43362.cpp

```diff
@@ -218,13 +218,19 @@ int ISM43362::connect(const char *ap, const char *passPhrase, ism_security_t ap_
         while (_parser.recv("%[^\n]\n", tmp)) {
             if (strstr(tmp, "OK")) {
                 _parser.flush();
+                _conn_status = NSAPI_STATUS_GLOBAL_UP;
+                _conn_stat_cb();
                 return NSAPI_ERROR_OK;
             }
             if (strstr(tmp, "JOIN")) {
                 if (strstr(tmp, "Failed")) {
                     _parser.flush();
+                    _conn_status = NSAPI_STATUS_DISCONNECTED;
```

Comment:
It should be `CONNECTING` as documented here: https://os.mbed.com/docs/mbed-os/v5.11/apis/network-interfaces.html (scroll down for the state diagram)

Reply:
> It should be CONNECTING

Done